back/app/services/permissions_service.rb

```diff
@@ -83,7 +83,7 @@ def remove_extras_actions(scope, actions = nil)
   def add_missing_actions(scope, actions = nil)
     missing_actions = missing_actions(scope, actions)
     permissions_hashes = missing_actions.map { |action| { action: action } }
-    Permission.create(permissions_hashes) { |p| p.permission_scope = scope }
+    Permission.create!(permissions_hashes) { |p| p.permission_scope = scope }
```

Comment:
I think the answer is no, but just to make sure: This wouldn't fail in expected cases? I mean, we're not counting on this to fail silently for normal operation?

Reply:
Alright!